### PR TITLE
Add StatusMessage related entities from the Ranen24's gem.

### PR DESCRIPTION
**Pull request by cmrd-senya**
This makes the [testbed](https://github.com/cmrd-senya/federation-testbed) pass with this gem.

### CODE REVIEW

**Comment by Raven24** [comment]
hi senya,
I saw your email, you're right to contribute to this repo, because apart from the gem I wrote ages ago there is no further integration of it into diaspora.
supertux picked up the ball and actually re-used much of my gem here and also integrated it into the diaspora code base.
so, I suppose contributing here has the biggest value for upstream ;)

**Comment by SuperTux88** [comment]
thanks for your pull-request :)

can you please format the entities like the existing in my gem: https://github.com/SuperTux88/diaspora_federation/blob/master/lib/diaspora_federation/entities/person.rb

* split the `module DiasporaFederation; module Entities` in two lines
* use `property :diaspora_id, xml_name: :diaspora_handle` for `diaspora_handle` because the official term is ID not handle. I try to remove `handle` as much as possible from the code.
* copy also the tests and make them run
* maybe also add documentation to the properties (haven't done this to all properties in profile.rb yet, so I can't force you, but it would be nice ;) )

**Comment by cmrd-senya** [comment]
Damn, I've broke it while clearing rubocop-git notifications.
Will fix soon.

File: spec/lib/diaspora_federation/entities/location_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module DiasporaFederation
+  describe Entities::Location do
+    let(:data) { FactoryGirl.attributes_for(:location_entity) }
+
+    let(:xml) {
```
Comment (diaspora-code-review) [inline spec/lib/diaspora_federation/entities/location_spec.rb]: Trailing whitespace detected.

**Comment by cmrd-senya** [comment]
Fixed.

**Comment by SuperTux88** [comment]
Thanks!

**Comment by Flaburgan** [comment]
Awesome @cmrd-senya thank you!

**Pull request merged**